### PR TITLE
290 style create campaign

### PR DESCRIPTION
**Pull request by TheBrooks**
- Connect #290 
- Styled the 3 step Create Campaign process
- Created the ability to optionally add a Waste Provider from supported waste providers (future feature to create a script to auto populate this table)
- Creating campaign now writes optional  property manager info, waste provider info and building size info to the model

### CODE REVIEW

File: client/src/components/CreateCampaignStep1.js [hunk 1]
```diff
@@ -20,49 +29,49 @@ const CreateCampaignStep1 = props => {
...
           </div>
-          <div className="form-group">
-            <label className="required">Campaign Name</label>
-            <div className="requiredtool" />
-            <input type="text" className="form-control campaign" name="campaignName" required />
+        )}
+        {!formattedAddress && (
+          <div className="text-center">
+            <p>
+              {'Add an address to start.'}{' '}
```
Comment (hoop71) [inline client/src/components/CreateCampaignStep1.js]: combine this trailing space bitmoji

File: client/src/components/CreateCampaignStep2.js [hunk 2]
```diff
@@ -1,63 +1,248 @@
...
-  const setOptionalInfo = async (e) => {
+class CreateCampaignStep2 extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.defaultProvider = {
+      _id: WASTE_PROVIDER_NOT_SET_ID,
+      name: 'Select your provider(Optional)',
+      phone: 'N/A',
+      email: 'N/A'
```
Comment (hoop71) [inline client/src/components/CreateCampaignStep2.js]: Suggest changing N/A to blank if waiting to populate from dropdown.

File: client/src/components/CreateCampaignStep2.js [hunk 3]
```diff
@@ -1,63 +1,248 @@
...
+          </Col>
+        </FormGroup>
+
+        <FormGroup>
+          <h2>Building Info:</h2>
+          <Col xs={12}>
+            <ControlLabel>THIS PROPERTY CONSISTS OF:</ControlLabel>
+          </Col>
+          <Col xs={2}>
+            <FormControl type="Number" name="buildingCount" placeholder={0} />
```
Comment (hoop71) [inline client/src/components/CreateCampaignStep2.js]: disable out of range numbers ie 0 < x < 100

File: client/src/components/CreateCampaignStep2.js [hunk 4]
```diff
@@ -1,63 +1,248 @@
...
+          </Col>
+          <Col xs={10}>BUILDING(S)</Col>
+          <Col xs={12}>
+            <ControlLabel>THIS BUILDING HAS APPROXIMATELY:</ControlLabel>
+          </Col>
+          <Col xs={2}>
+            <FormControl type="number" name="unitCount" placeholder={0} />
+          </Col>
+          <Col xs={10}>TOTAL UNITS</Col>
```
Comment (hoop71) [inline client/src/components/CreateCampaignStep2.js]: same as above

File: client/src/stylesheets/components/_NewCampaign.scss [hunk 5]
```diff
@@ -1,30 +1,75 @@
...
+  .create-campaign-control-label {
+    display: inline;
+  }
+  .form-group {
+    div {
+      padding-left: 5px;
+      padding-right: 5px;
+    }
+    .readonly {
+      background-color: rgba(200, 200, 200, 0.9);
```
Comment (hoop71) [inline client/src/stylesheets/components/_NewCampaign.scss]: Sass variable?

File: client/src/components/CreateCampaignStep1.js (hunk 1 above)
Comment (TheBrooks) [reply client/src/components/CreateCampaignStep1.js]: ok

File: client/src/components/CreateCampaignStep2.js (hunk 2 above)
Comment (TheBrooks) [reply client/src/components/CreateCampaignStep2.js]: ok

File: client/src/components/CreateCampaignStep2.js (hunk 3 above)
Comment (TheBrooks) [reply client/src/components/CreateCampaignStep2.js]: ok

File: client/src/components/CreateCampaignStep2.js (hunk 4 above)
Comment (TheBrooks) [reply client/src/components/CreateCampaignStep2.js]: ok

File: client/src/stylesheets/components/_NewCampaign.scss (hunk 5 above)
Comment (TheBrooks) [reply client/src/stylesheets/components/_NewCampaign.scss]: chose to darken($mainwhite, 15%) instead

**Pull request merged**